feature/player/src/main/AndroidManifest.xml

```diff
@@ -1,13 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android">
 
     <application android:usesCleartextTraffic="true">
         <activity
             android:name=".PlayerActivity"
-            android:configChanges="orientation|screenSize|smallestScreenSize|screenLayout|uiMode|keyboard|keyboardHidden|navigation"
```

Comment:
@Hacker437 What's the reason behind removing these config changes

Reply:
The reason for this when Pip mode we don't wanna recreate activity when user uses another app like Instagram

Reply:
To avoid re-creating activity when users interact with keyboard-based apps, you should include the keyboard option in android:configChanges rather than removing it.